tstest/integration/vms/doc.go

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package vms does VM-based integration/functional tests by using
+// cloud-init enabled images.
```

Comment:
drop cloud-init part. That won't be a requirement in the future. That's not key to what this does necessarily. The main point is that it drives VMs with qemu.